client/overview/task-list/po-tasks.js

```diff
@@ -0,0 +1,126 @@
+/** @format **/
+
+/**
+ * External dependencies
+ */
+import moment from 'moment';
+import { addQueryArgs } from '@wordpress/url';
+
+/**
+ * Internal dependencies.
+ */
+import strings from './strings';
+
+const PROGRESSIVE_ONBOARDING_KYC_TPV_LIMIT = 5000;
+
+export const getVerifyBankAccountTask = ( {
+	poEnabled,
+	poComplete,
+	tpv,
+	firstPaymentDate,
+	createdDate,
+} ) => {
+	if ( ! poEnabled ) {
+		return null;
+	}
+	if ( poComplete ) {
+		return null;
+	}
```

Comment:
_Nitpick:_ We should also check for `createdDate`, because with an empty date the task will be rendered. And we can group the three conditions together.

```suggestion
	if ( ! poEnabled || poComplete || ! createdDate ) {
		return null;
	}
```